cinema/serializers.py

```diff
@@ -68,13 +78,62 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available"
         )
 
 
+class TicketDetailSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ticket
+        fields = ["seat", "row"]
+
+
 class MovieSessionDetailSerializer(MovieSessionSerializer):
     movie = MovieListSerializer(many=False, read_only=True)
+    genres = GenreSerializer(many=True, read_only=True)
```

Comment:
The `genres` field in `MovieSessionDetailSerializer` seems misplaced. `MovieSession` typically does not have a direct relationship with `genres`. Consider removing this field or ensuring it aligns with your model relationships.